packages/react-data-grid/src/Grid.js

```diff
@@ -64,11 +63,6 @@ const Grid = createReactClass({
     overScan: PropTypes.object
   },
 
-  mixins: [
```

Comment:
`GridScrollMixin` is only used by the `Grid` component so moved the logic into the `Grid` component

`DOMMetrics.MetricsComputatorMixin` defines `childContextTypes/getChildContext` to implicitly pass `metricsComputator` to all the descendant components. `ReactDataGrid`  (Parent) component is using the same mixin and acts as the context provider so `Grid` component does not need to add it. It can be safely removed

https://reactjs.org/docs/context.html